core/services/ocr2/plugins/directrequestocr/test/internal/testutils.go

```diff
@@ -57,7 +57,7 @@ func SetOracleConfig(t *testing.T, owner *bind.TransactOpts, oracleContract *ocr
 			MaxQueryLengthBytes:       10_000,
 			MaxObservationLengthBytes: 10_000,
 			MaxReportLengthBytes:      10_000,
-			MaxRequestBatchSize:       5,
+			MaxRequestBatchSize:       4,
```

Comment:
@bolekk had to lower this by 1 to get the integration test to pass now,
We could raise the report `gasLimit`, but I didn't see an obvious way.